packages/runtime/container-runtime/src/test/gc/garbageCollection.spec.ts

```diff
@@ -626,6 +629,7 @@ describe("Garbage Collection Tests", () => {
 				);
 
 				// Add reference from node 2 to node 3 and validate that revived event is logged.
+				defaultGCData.gcNodes[nodes[2]] = [nodes[3]];
```

Comment:
This seems like it doesn't do anything, but is it just to be consistent when "adding a reference", to update both this and call `addedOutboundReference`?

Reply:
The reference was removed above in line 606. This adds it back. It works without this too because we reset the unreferenced timestamp in this scenario resulting the node not being in inactive+ stage. So, a revived event is logged.